applications/electron/package.json

```diff
@@ -5,6 +5,7 @@
   "productName": "Theia IDE",
   "version": "1.67.100",
   "main": "scripts/theia-electron-main.js",
+  "desktopName": "org.eclipse.theia.desktop",
```

Comment:
ah well, I see you've also hit the issue now. Ref https://github.com/eclipse-theia/theia-ide/pull/635#discussion_r2649106295

Reply:
Please double check if that can be added here safely, or whether that breaks assumptions made by AppImage or deb builds. (Alternately this could be patched in the flatpak build itself)